core/livepeernode.go

```diff
@@ -142,14 +147,18 @@ func (n *LivepeerNode) GetBasePrice(b_eth_addr string) *big.Rat {
 	n.mu.RLock()
 	defer n.mu.RUnlock()
 
-	return n.priceInfo[addr]
+	return n.priceInfo[addr].Value()
```

Comment:
```suggestion
	price := n.priceInfo[addr]
	if price != nil {
		return price.Value()
	}
	return nil
```

I think you need to check if n.priceInfo[addr] is not nil.